src/content/blogs/onelab-launch-at-linaro-connect-madrid-2024.mdx

```diff
@@ -0,0 +1,54 @@
+
+In the afternoon, we showed ONELab in action and things became more focused with a Plugfest followed by an Advisory meeting. Don Harbin and Daniel Thompson from Linaro (who are leading the partner experience and technical support), introduced the plan to release ONELab officially in Q4. 
+
+![ONELab - IoT HW ODM Onboarding](/linaro-website/images/blog/ONELab-IoT-HW-ODM-Onboarding_njhryv.png)
+
+Among the plans discussed was migrating from testing within Linaro’s Lab to the desired virtual presence of the partner devices locally in their Labs. Systems early in their life cycle are usually in short supply and expensive, whilst prone to frequent updates, allowing early participation in the lab. Leaving the devices to be tested (DUTs) with the engineers who are experts in its bring up can significantly accelerate discovering and resolving issues uncovered.
+
+![ONELab kickoff event](/linaro-website/images/blog/ONELab-kickoff-event_yh6ylr.png)
+*ONELab kickoff event <br />*
+
+A groundbreaking highlight was the ease with which diverse partner DUTs were attached to the service. Both TI and Qualcomm had brought their hardware to the event and, in just half an hour, checks were completed and ready to run. This feature represents a key change for owners of large board farms, who can confirm that onboarding is currently fraught with pitfalls, and enablement takes days to months to accomplish. Instead with ONELab time for this operation is sensitively reduced and accomplished with extreme ease.
+
+Enabling the Qualcomm RB2 | Enabling the TI SK-AM62B
+:---:|:---:
+![](/linaro-website/images/blog/enabling-qualcomm-rb2_pxtv96.jpg) | ![](/linaro-website/images/blog/enabling-ti-sk-am62b_t5ulnz.jpg)
+
+The Linaro Automation Appliance (LAA) was able to control both boards' critical features, such as UART, Ethernet, and power and reset, allowing them to remotely participate in the ONELab.
+
+The [LAA](https://resources.linaro.org/en/resource/fwWCArmwQyjvLgnFMdZwwT) provides complete remote access to a DUT at the production scale, including full recovery of bricked devices using the widely adopted [LAVA CI model](https://docs.lavasoftware.org/lava/lava_ci.html). With this setup, all further enablement activities can be performed remotely using the appliance's remote bring-up tools.
+
+We plan to prove this enablement by shipping the boards directly to our lab in Cambridge and assigning the remaining bring-up tasks to one of our non-Cambridge employees.
+
+![](/linaro-website/images/blog/onelab-powered-by-linaro_lssvkh.jpg)
```

Comment:
alt text shouldn't be blank unless it's a background image or something like that